types/src/client.rs

```diff
@@ -56,6 +56,19 @@ pub struct SubscriptionMessage {
 	pub send_back: oneshot::Sender<Result<(mpsc::Receiver<JsonValue>, SubscriptionId), Error>>,
 }
 
+/// OnNotification message.
+#[derive(Debug)]
+pub struct OnNotificationMessage {
+	/// Request ID of the subscribe message.
+	pub req_id: u64,
```

Comment:
This seems needless given that we only care about that `method_name`

I assume this is a workaround to work to work how current subscriptions are handled in the `RequestManager`.

Reply:
Yep, I will refactor out